test/unit/rubygem_test.rb

```diff
@@ -895,6 +895,28 @@ class RubygemTest < ActiveSupport::TestCase
     end
   end
 
+  context ".mfa_required_from_downloads" do
+    should "not return gems with fewer downloads than the required threshold" do
+      rubygem = create(:rubygem)
+      GemDownload.increment(
+        Rubygem::MFA_REQUIRED_THRESHOLD,
+        rubygem_id: rubygem.id
+      )
+
+      assert_empty Rubygem.mfa_required_from_downloads
+    end
+
+    should "return gems with more downloads than the required threshold" do
+      rubygem = create(:rubygem)
+      GemDownload.increment(
+        Rubygem::MFA_REQUIRED_THRESHOLD + 1,
+        rubygem_id: rubygem.id
+      )
+
+      refute_empty Rubygem.mfa_required_from_downloads
```

Comment:
I'd add an assertion to verify that `rubygem` is within the collection of `Rubygem.mfa_required_from_downloads`.

Reply:
Good idea, I'm going to remove the original since it should be redundant when I add the includes? check